src/pages/Sketch/components/Toolbox/Toolbox.js

```diff
@@ -108,18 +116,35 @@ function Toolbox({
           />
         </div>
       </Feature>
+
       {type !== "text" && (
-        <Feature classname={styles.sliderWrapper} title={`Stroke Roughness`}>
+        <Feature classname={styles.sliderWrapper} title={`Roughness`}>
           <input
             className={styles.slider}
             type="range"
             min={0}
             max={5}
+            step={0.1}
             value={roughness}
             onChange={e => setRoughness(e.target.value)}
           />
         </Feature>
       )}
+
+      {type !== "text" && (
```

Comment:
Rather than checking this condition (`type !== "text"`) for every single feature, it's better to club every `Feature` (which falls under this condition) together under a single check.

```
// Just an ex.
{
    `type !== "text && (
         <><Feature ... /> <Feature #2 .... /> ... {and so on} </>
         )
}
```

Reply:
@iamdarshshah thanks for your detailed examples.😄😄